tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -871,4 +871,32 @@ public function testHasColumnReservedName()
         $this->assertTrue($tableQuoted->hasColumn('value'));
 
     }
+
+    public function testGetColumnTypes()
```

Comment:
If the intent of this test is simply to make sure that `enum` and `set` are in the column types, that should be the only thing tested. As currently written, this test is fragile because adding new base types will cause it to fail.